FALSEHOODS.md

```diff
@@ -235,3 +235,10 @@ surprising how many false assumptions programmers continue to make about them.
     during, or after the actual implementation of numbering plan changes in the
     real world. The actual date on which a phone number range becomes active may
     not always adhere to official announcements.
+
+1.  **Phone numbers are numbers**
```

Comment:
I agree about the positioning, so I'll move it. I prefer my title though - it's short and snappy, like in the list that (I assume) inspired this file: https://www.kalzumeus.com/2010/06/17/falsehoods-programmers-believe-about-names/